src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.HandleArray.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Text.Json.Serialization.Policies;
+
+namespace System.Text.Json.Serialization
+{
+    public static partial class JsonSerializer
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static void HandleStartArray(
```

Comment:
Remove the `AggressiveInlining` attributes used in such large methods.

Reply:
In all of the places where I use it the method is called only from one location. However, I will remove them and let the compiler decide.